rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/CommandSet.java

```diff
@@ -680,6 +682,23 @@
                                                                           .commandClass(AddNodeCommand.class)
                                                                           .entry();
 
+    public static final CommandSet.Entry NS_DEFINE = CommandSetEntryBuilder.newInstance()
+                                                                           .opt("dfns")
+                                                                           .longOpt("definens")
+                                                                           .description("Define new node source")
+                                                                           .hasArgs(true)
+                                                                           .numOfArgs(1)
+                                                                           .hasOptionalArg(true)
+                                                                           .argNames("node-source [nodes-recoverable]")
+                                                                           .jsCommand("definens(node-source[,nodes-recoverable])")
+                                                                           .commandClass(DefineNodeSourceCommand.class)
+                                                                           .entry();
+
+    /**
+     * @deprecated  As of release 7.37, replaced by
+     * {@link CommandSet#NS_DEFINE} and {@link CommandSet#NS_DEPLOY}
+     */
+    @Deprecated
     public static final CommandSet.Entry NS_CREATE = CommandSetEntryBuilder.newInstance()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)